frontend/src/lib/components/icons.stories.tsx

```diff
@@ -59,3 +60,55 @@ export function Icons(): JSX.Element {
         />
     )
 }
+
+export function IconWithCountBubble(): JSX.Element {
+    const betweenOneAndTwenty = Math.floor(Math.random() * (20 - 1 + 1)) + 1
```

Comment:
Can you remove the randomness?

Theoretically this breaks the component's purity, which React assumes we keep. Now every render renders something different, and it's not guaranteed when anything will be rendered.

Plus it'll be another thing to fix, once we introduce visual regression testing.